jdaviz/core/template_mixin.py

```diff
@@ -453,6 +455,9 @@ def __init__(self, app, tray_instance=False, **kwargs):
 
         super().__init__(app=app, **kwargs)
 
+        if self._plugin_name is not None:
+            self.plugin_description = get_plugin_description(self.app.config, self._plugin_name)
+
```

Comment:
for downstream plugins that won't have access to write a case in `get_plugin_description`, will this crash?  Will they be able to just manually set `plugin_description` in the init?

Reply:
im trying to figure this out, setting it in the plugin init is not working at the moment. the change is not reflected in the app. I think this might be because its set during 'load_configuration', so it won't change unless that's run again.

Reply:
you can set it by doing something like 'self.app.state.tray_items[0]['description'] = 'new description'

Reply:
ok, but I think we want to be able to set it in the plugin's `__init__`, that should be called before the `tray_items` is populated, right?  Changes afterwards (based on a change in data, or something like that) probably can't be expected to propagate right now without significant refactoring of how `tray_items` works (which I don't think is worth the effort now).

Reply:
done